src/content/ui-api/management-api/authorization/index.md

```diff
@@ -0,0 +1,246 @@
+
+**Grant access to the organization**. For every subject bound to any role in an organization's namespace, we ensure that the subject also has `get` permission to the `Organization` resource defining that organization. (Why? As this allows clients like our web UI to detect which organizations a user has access to, without requiring `list` permissions for organizations.)
+
+## Typical use cases {#typical-use-cases}
+
+For the purpose of this documentation article we'll introduce a few example cases and then explain how to configure access for them, starting with the least permissions and ending with the highest privileges. Chances are that you can use them as a starting point for your own requirements.
+
+1. **Read-only user**: allowed to "browse" most resources in specific organizations.
+
+2. **App developer**: allowed to install, configure, upgrade and uninstall certain apps in/from workload clusters of a certain organization. In order to discovers clusters and navigate the web UI, users of this type also require read permission to various resources like clusters, node pools, releases, and app catalogs.
+
+3. **Organization admin**: users who have full permissions to resources belonging to a specific [organization]({{< relref "/general/organizations" >}}).
+
+4. **Admin**: a type of user that has permission to create, modify, and delete most types of resources in the management cluster.
+
+Next we show how to configure access for these example cases. Please take into account these remarks for all example manifests:
+
+- You can set the resource names (for `RoleBinding` and `ClusterRoleBinding` resources) to whatever suits you and is available.
+- Make sure to replace `GROUPNAME` with the exact name of your group as defined in your identity provider.
+- To assign a user instead of a group, replace `kind: Group` with `kind: User` in the subject entry and set `name` to ... TODO: email or other handle?
+
+### Configure access for read-only users {#configure-read-only}
+
+A read-only user in the sense of the Management API is one that can discover or, in the Giant Swarm web UI, browse resources in order to find out about existing clusters and their configuration, as well as apps installed and available.
+
+To enable a group or user for read access to resources of one organization, you only have to bind them to these two cluster roles within the namespace of the organization:
+
+- `read-all`
+- `read-in-cluster-ns`
+
+More details on these roles are given in the [pre-defined roles](#pre-defined-roles) section.
+
+The manifest below shows how the according role bindings could be created.
+
+```yaml
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  creationTimestamp: null
+  name: bind-read-all-to-group
```

Comment:
I think we have to include the `namespace` here as well in order to create this role binding within an organization namespace